### DISCUSSION TITLE
No puedo subir scorm a lecciones se queda en 0%.

### DISCUSSION THREAD

**Discussion by ossiof** [chamilo/chamilo-lms#5457] [General]
No puedo subir scorm a lecciones se queda en 0%. Tampoco importar una copia de seguridad. Tampoco modificar las categorías. ¿Hay un problema de configuración de Chamilo?

**Comment by Kaneda-1** [comment]
Hola @ossiof un poquito de información adicional sería aconsejable que proporcionaras ;-)

¿Es un campus propio?  
¿tienes acceso a los logs de tu servidor? 
¿Qué tamaño en megas tiene el fichero SCORM? 
¿Qué tamaño en megas  tiene la copia de seguridad? 
¿Tiene espacio asignado el curso en el que lo estás subiendo?

Un saludo.

**Reply by ossiof** [reply]
Gracias Kaneda-1.
Lo mismo me ocurre cuando instalo Chamilo en modo local.
He contratado un Hosting con dominio propio y a través del auto instalador del cpanel se hizo la instalación.
El fichero Scorm no tiene más de 4.1 MB. No es problema del tamaño. Son archivos que estoy reutilizando de otros cursos brindados.
Esta con el espacio por defecto.
No es problema de espacio tampoco porque recién estoy subiendo archivos.
Tampoco puedo realizar copias del curso.
Muchas gracias por tu tiempo. Espero puedas ayudarme.
Saludos.

**Comment by Kaneda-1** [comment]
Hola. ¿Qué versión de php estás utilizando? PHP 8 o superior? 
Si es así revisa https://github.com/chamilo/chamilo-lms/discussions/5287#discussioncomment-8915228
Un saludo.

**Comment by Kaneda-1** [comment]
Hola.

He probado en una demo de Chamilo con el último código de github https://github.com/chamilo/chamilo-lms/tree/1.11.x (tienes que realizar un composer update para generar las carpetas web y vendor) y no me ocurre lo que indicas al subir contenido SCORM.

¿Qué versión de Chamilo estás usando la que descargas de https://chamilo.org/en/download/?

Aquí también lo comentan https://github.com/chamilo/chamilo-lms/discussions/5220

Un saludo.

**Comment by ossiof** [comment]
¿Es posible que instalando versiones anteriores de Chamilo se resuelva  estos problemas? Muchas gracias. 

Yahoo Mail: busca, organiza, toma el control de tu buzón 
 
  El mar., 30 de abr. de 2024 a la(s) 7:59, ***@***.***> escribió:   


Hola.

He probado en una demo de Chamilo con el último código de github https://github.com/chamilo/chamilo-lms/tree/1.11.x (tienes que realizar un composer update para generar las carpetas web y vendor) y no me ocurre lo que indicas al subir contenido SCORM.

¿Qué versión de Chamilo estás usando la que descargas de https://chamilo.org/en/download/?

Aquí también lo comentan #5220

Un saludo.
